Sources/TunnelKitWireGuardManager/WireGuardDataCount.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  WireGuardDataCount.swift
+//  
+//
+//  Created by Yevgeny Yezub on 17/11/23.
+//
+
+import Foundation
+/// A pair of received/sent bytes count.
+public struct WireGuardDataCount: Equatable {
+
+    /// Received bytes count.
+    public var bytesReceived: UInt
+
+    /// Sent bytes count.
+    public var bytesSent: UInt
+
+    public init(_ received: UInt, _ sent: UInt) {
+        self.bytesReceived = received
+        self.bytesSent = sent
+    }
+}
+
+extension WireGuardDataCount {
+    public init?(from string: String?) {
```

Comment:
Please don't fall back this much, guard against optional String in caller instead.

```suggestion
    public init?(from string: String) {
```

Reply:
Noted and fixed